tests/3d_brain.py

```diff
@@ -0,0 +1,87 @@
+from xdesign.grid import *
```

Comment:
We use [`nosetests`](http://nose.readthedocs.io/en/latest/) for our test scripts so that they can be run automatically on `Travis` and so that debug logging is automatically set when they are run. The quick and dirty is that all tests should be in files with the word `test` in the filename and each test should be a function within that file with `test` in the title.